client/packages/coldchain/src/Equipment/DetailView/DetailView.tsx

```diff
@@ -22,15 +22,28 @@ import { AssetFragment, useAssets } from '../api';
 import { StatusLogs } from './Tabs/StatusLogs';
 import { Documents } from './Tabs/Documents';
 import { ActivityLogList } from 'packages/system/src';
+import { useLocation } from 'packages/system/src';
```

Comment:
importing from `packages/..` will break jest if the file is used in a test. Can you also remove the above line, which is probably my mistake

```suggestion
import { ActivityLogList, useLocation } from '@openmsupply-client/system';
```